src/pages/Antiquities/Antiquities.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Accordion, Avatar, Container, Group, Text, Title } from '@mantine/core';
+import locationsJson from '@sources/locations.json';
+import antiquitiesJson from '@sources/antiquities.json';
+import classes from './Antiquities.css';
+import { AntiquityCard } from '@/components/AntiquityCard/AntiquityCard';
+import { Antiquity, Location } from '@/@typings/database-types';
+
+export function Antiquities() {
+  const locations: Location[] = locationsJson;
+  const antiquities: Antiquity[] = antiquitiesJson;
+
+  return (
+    <Container size="sm" className={classes.container}>
+      <Title order={1} className={classes.title}>
+        Antiquities
+      </Title>
+      <Accordion>
+        {locations.map((location) => (
```

Comment:
Accordion in a accordion in a accordion does not really look like a good UX.

Why won't us use Selects for 2 of those or all instead and display a content?